install-script.sh

```diff
@@ -69,7 +69,10 @@ swift build -c release --arch arm64 --arch x86_64
 cd .build/apple/Products/Release
 
 echo "** Install..."
-cp "$(xcode-select -p)"/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/macosx/lib_InternalSwiftSyntaxParser.dylib . 
+if [[ ! -e ./lib_InternalSwiftSyntaxParser.dylib ]]; then
+    echo "** Copy lib_InternalSwiftSyntaxParser.dylib from Xcode toolchain..."
+    cp "$(xcode-select -p)"/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/macosx/lib_InternalSwiftSyntaxParser.dylib . 
```

Comment:
Is this fallback necessary?
In almost all cases, this will not work and the original issue will be hidden.
This script should fail here.

Reply:
Thank you for your review.

> Is this fallback necessary?

In my understanding, if swift version is under `5.6`, `lib_InternalSwiftSyntaxParser` will not be prepared in resolving by SPM.
so in case user's swift environment is under swift 5.6, I didn't delete option to copy from xcode toolchain.

Reply:
In my understand, this script is used for distribution.

For legacy swift user, artifacts exist in release page. Also they can build mockolo theirself ( this is guaranteed by https://github.com/uber/mockolo/blob/master/.github/workflows/builds.yml )
Some escape hatches exist so not needs to consider backward compatibility I think.

Reply:
Ah, you are correct. `install-script.sh` is not an install option but distribution way.
I removed the fallback.